.github/workflows/reusable-test-latest-deps.yml

```diff
@@ -63,9 +75,8 @@ jobs:
           GE_CACHE_PASSWORD: ${{ secrets.GE_CACHE_PASSWORD }}
         with:
           arguments: >
-            test
```

Comment:
when we migrated to test suites we stopped running some of the latest deps tests because they now depend on `check` instead of `test`, that is why this pr also includes some changes to tests

Reply:
😱 great catch